City-of-Helsinki/drupal-helfi-platform-config
# UHF-3169 Contact cards listing

# TASKNAME [UHF-3169](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3169)

## What was done
* Created a contact card listing paragraph which makes it possible to list as many contacts as a user wants. Contacts are manually updated. Each contact can have a maximum of 6 different social media links (up for the user to select the correct icons etc). The paragraph was added to standard pages upper and lower content regions and well as for both content regions in tpr unit.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh` in project root folder
* Update the Helfi Platform config 
    * run `make shell` in the root folder
    * run in the shell `composer require drupal/helfi_platform_config:dev-UHF-3169-contact-cards-listing`
    * and lastly `drush updb; drush cr`

## How to test
* Test the feature by creating or editing a standard page and make sure the paragraph can be added to both upper and lower content regions. Also test that there is a limit of 6 social media links per contact. Check that the paragraph can also be found from tpr units upper and lower content regions.

(note that styles etc are not applied in this pr and paragraph won't look like the design)
